sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventDataBatch.java

```diff
@@ -102,6 +101,8 @@ public int getSizeInBytes() {
     /**
      * Tries to add an {@link EventData event} to the batch.
      *
+     * <p>This method is thread-unsafe</p>
```

Comment:
@anuchandy Hi Anu, Do you think we need to add more infomation to let user know like:
`<p>This method is thead-unsafe. Please use lock if you want to add events in multi-thread.</p>` 

And do we need add some information in README.md because I saw the examples also use `tryAdd()`?

Reply:
Hi Kun, that looks good, slightly reworded if you're ok with that ```This method is not thread-safe; make sure to synchronize the method access when using multiple threads to add events. ```

Agree with you, it's a good idea to add a note in the readme as well.

Reply:
Thanks for the suggestions. Already updated the code.